docs/modules/integrate/pages/get-started-with-vertx.adoc

```diff
@@ -0,0 +1,460 @@
+= Get started with Vert.x
+
+This tutorial helps you integrate Vert.x with Hazelcast and use Hazelcast for distributed session management and other distributed data structures.
+
+In this tutorial, you will
+- start with a simple Vert.x Hello World application
```

Comment:
```suggestion
In this tutorial, you will

- start with a simple Vert.x Hello World application
```

Not sure if I introduced this spacing error but the bullets not displaying in the preview